openshift_images/using_images/images-other-jenkins-agent.adoc

```diff
@@ -0,0 +1,45 @@
+[id="images-other-jenkins-agent"]
+= Jenkins agent
+include::modules/common-attributes.adoc[]
+:context: images-other-jenkins-agent
+toc::[]
+
+{product-title} provides three images that suitable for use as Jenkins agents:
+the *_Base_*, *_Maven_*, and *_Node.js_* images.
+
+The first is a base image for Jenkins agents:
+
+* It pulls in both the required tools, headless Java, the Jenkins JNLP client,
+   and the useful ones including `git`, `tar`, `zip`, and `nss` among others.
+* It establishes the JNLP agent as the entrypoint.
+* It includes the `oc` client tooling for invoking command line operations from
+   within Jenkins jobs.
+* It provides Dockerfiles for both CentOS and Red Hat Enterprise Linux (RHEL) images.
```

Comment:
it is now RHEL7and "localdev" for those docker files.... localdev is set up for building when you don't have RHEL subscriptions